includes/Modules/Analytics_4.php

```diff
@@ -92,9 +95,13 @@ public function register() {
 	 * @return array List of Google OAuth scopes.
 	 */
 	public function get_scopes() {
-		return array(
+		$scopes = array(
 			Analytics::READONLY_SCOPE,
 		);
+		if ( Feature_Flags::enabled( 'gteSupport' ) ) {
+			$scopes[] = Google_Service_TagManager::TAGMANAGER_READONLY;
```

Comment:
I like the idea here, but this should probably use the same value as defined by the Tag Manager module (see the same method of the Tag Manager module). I think we could refactor to use these constants in a follow-up issue, however.